LibOS/shim/src/shim_malloc.c

```diff
@@ -358,11 +208,41 @@ void * calloc (size_t nmemb, size_t size)
 }
 extern_alias(calloc);
 
+#if 0 /* Temporarily disabling this code */
```

Comment:
Is this going to be a permanent disabling?  Should it just be deleted?

Reply:
realloc is a nice-to-have feature, IMO it's worth reenabling after the migration problem gets fixed.